apis/clusters/v1beta1/redis_webhook.go

```diff
@@ -63,10 +64,13 @@ func (r *Redis) Default() {
 
 	for _, dataCentre := range r.Spec.DataCentres {
 		dataCentre.SetDefaultValues()
-	}
-
-	redislog.Info("default values are set", "name", r.Name)
 
+		if dataCentre.ReplicationFactor > 0 && dataCentre.NodesNumber == 0 {
+			dataCentre.NodesNumber = dataCentre.MasterNodes * dataCentre.ReplicationFactor
+		} else if dataCentre.NodesNumber > 0 && dataCentre.ReplicationFactor == 0 {
+			dataCentre.ReplicationFactor = dataCentre.NodesNumber / dataCentre.MasterNodes
+		}
+	}
```

Comment:
If the replication factor and nodes number are equal to 0, I assume that we create master nodes only, am I right?

Reply:
Yes, you are.